cypress_test/integration_tests/multiuser/calc/repair_document_spec.js

```diff
@@ -3,7 +3,7 @@
 var helper = require('../../common/helper');
 var calcHelper = require('../../common/calc_helper');
 
-describe('Repair Document', function() {
+describe('Repair Document',function() {
```

Comment:
The space was actually better, and consistent with the other cases.

Reply:
+1, better to not change unrelated lines